Userland/Libraries/LibWeb/HTML/HTMLOrSVGElement.idl

```diff
@@ -1,3 +1,5 @@
+#import <HTML/DOMStringMap.idl>
```

Comment:
Note: I left this there. as https://github.com/LadybirdBrowser/ladybird/pull/2036 added this file without this. I am a fan of explicit imports, it also worked before this, since everyone that imports this file also imports `DOMStringMap` in some implicit way, and this is a mixin, so can't used directly.

Reply:
Thanks! I indeed forgot to add it.